SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -6,19 +6,20 @@ namespace SeleniumPracticalExercise.PageObjects
 {
     public class EmployeeInformationPage : BasePageLocal
     {
-        
+        private readonly By _actualFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
+        private readonly By _actualIDLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");
```

Comment:
Remember, variables should have camel case names... even common acronyms, e.g. "ID" should be "Id"... "_actualIdLocator".

Reply:
renamed to idResultsLocator